ocaml/default.nix

```diff
@@ -107,6 +107,18 @@ with oself;
     };
   });
 
+  benchmark = osuper.buildDunePackage {
```

Comment:
this should have been `buildDunePackage` (from `self`). If we ever override `buildDunePackage`, we'd want it to be picked up.

Reply:
Sorry :grimacing: